Tests fails on Fedora rawhide

```
+ make test docker_run=
bash ./clitest --first --progress none test.md
--------------------------------------------------------------------------------
[FAILED #133, line 1164] ./clitest --list-run test/inline-match-egrep.sh | sed 's/^\(#1[56].\)[A-Z]*/\1?/'
@@ -1,3 +1,5 @@
+grep: warning: stray \ before t
+grep: warning: stray \ before n
 #1	OK	echo 'abc123'                 
 #2	OK	echo 'abc123'                 
 #3	OK	echo 'abc123'                 
--------------------------------------------------------------------------------
make: *** [Makefile:28: test-bash] Error 1
```

Full log: https://kojipkgs.fedoraproject.org//work/tasks/3914/100143914/build.log


Some versions:
```
bash                          5.2.15-3.fc38   
coreutils                     9.3-1.fc39      
diffutils                     3.9-4.fc39      
findutils                     1:4.9.0-4.fc39  
gawk                          5.1.1-5.fc38    
grep                          3.9-1.fc39      
info                          7.0.3-1.fc39    
patch                         2.7.6-21.fc39   
sed                           4.8-12.fc38     
```

--- a/test/inline-match-egrep.sh
+++ b/test/inline-match-egrep.sh
@@ -42,7 +42,7 @@ $ printf ' \t  \t\t   \n'       #=> --egrep ^ 	  		   $
 # egrep regexes. You'll need to test in your system if that's the
 # case. I recommend using a literal tab to avoid problems.
 
-$ printf 'may\tfail'            #=> --egrep ^may\tfail$
+$ printf 'may\tfail'            #=> --egrep ^may\\tfail$
 $ printf 'may\tfail'            #=> --egrep ^may[\t]fail$
 $ printf 'will\tmatch'          #=> --egrep ^will	match$
 
@@ -51,7 +51,7 @@ $ printf 'will\tmatch'          #=> --egrep ^will	match$
 # These tests will fail:
 
 $ printf 'will\nfail'           #=> --egrep will.*fail
-$ printf 'will\nfail'           #=> --egrep will\nfail
+$ printf 'will\nfail'           #=> --egrep will\\nfail
 
 # If one line of a multiline results matches, the test is OK
 

